fabric-biomes-v1/src/main/java/net/fabricmc/fabric/api/biomes/v1/OverworldBiomes.java

```diff
@@ -107,4 +107,30 @@ public static void addBiomeVariant(Biome replaced, Biome variant, double chance,
 	public static void setRiverBiome(Biome parent, Biome river) {
 		InternalBiomeData.setOverworldRiverBiome(parent, river);
 	}
+
+	/**
+	 * Adds the biome as a large edge biome of the parent biome, with the specified weight.
+	 * Large edges are around the same size as hills biomes.
```

Comment:
I disagree, I think that mentioning that they add the same size as an edge biome helps when visualizing them. Perhaps we could mention that they are the same size as jungle edge biomes (at least I think)

Reply:
My point was that the Hills biome is never used as an edge biome, so it wasn't a good visualization. Mentioning an edge biome that is actually an edge biome would work.

Reply:
But I do not think that the hills part should be removed because at the end of the day, the hills and sub biomes will be exactly the same size as the large edge biome. For people already knowledgeable with the general size of a hills biome, this comparison does help (it certainly helped me)

Reply:
I think having both would be a good idea because of the reasons that you stated, we just need to be 100% sure that the jungle edge biomes are in fact the same size.

Reply:
I still fail to see how saying the size of a biome that wraps around another biome is the same as a biome that's inside of another biome; that doesn't tell me anything about the width of a large edge biome.

Reply:
Jungle edge biomes are only a large edge when bordering swamps

Reply:
any progress on this discussion?